src/app/WriteClient.cpp

```diff
@@ -328,6 +449,8 @@ CHIP_ERROR WriteClient::OnMessageReceived(Messaging::ExchangeContext * apExchang
     {
         err = StatusResponse::ProcessStatusResponse(std::move(aPayload), status);
```

Comment:
This logic is not correct, please check the spec. Even in a chunked write, you get back `WriteResponseMessage` for every `WriteRequestMessage`. A status response is only sent back if an error has occurred.

Reply:
Fixed